src/tests/functional/plugin/cpu/single_layer_tests/transpose.cpp

```diff
@@ -1,323 +1,327 @@
-//                                 ::testing::ValuesIn(inputOrderPerChannels5D),
-//                                 ::testing::ValuesIn(netPrecisionsPerChannels),
-//                                 ::testing::Values(CommonTestUtils::DEVICE_CPU),
-//                                 ::testing::Values(additional_config),
-//                                 ::testing::Values(cpuParams_ndhwc)),
-//                         TransposeLayerCPUTest::getTestCaseName);
-//
-//INSTANTIATE_TEST_SUITE_P(smoke_dynamicShapes5D_PermutePerChannels, TransposeLayerCPUTest,
-//                         ::testing::Combine(
-//                                 ::testing::ValuesIn(dynamicInputShapes5D),
-//                                 ::testing::ValuesIn(inputOrderPerChannels5D),
-//                                 ::testing::ValuesIn(netPrecisionsPerChannels),
-//                                 ::testing::Values(CommonTestUtils::DEVICE_CPU),
-//                                 ::testing::Values(additional_config),
-//                                 ::testing::Values(CPUSpecificParams{})),
-//                         TransposeLayerCPUTest::getTestCaseName);
-//
-//} // namespace
-//} // namespace CPULayerTestsDefinitions
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+// Since the Transpose ngraph operation is converted to the transpose node, we will use it in the transpose test
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+
+typedef std::tuple<
+        InputShape,                    // Input shapes
+        std::vector<size_t>,                // Input order
+        InferenceEngine::Precision,         // Net precision
+        std::string,                        // Target device name
+        std::map<std::string, std::string>, // Additional network configuration
+        CPUSpecificParams> TransposeLayerCPUTestParamSet;
+
+class TransposeLayerCPUTest : public testing::WithParamInterface<TransposeLayerCPUTestParamSet>,
+                              public ov::test::SubgraphBaseTest, public CPUTestsBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<TransposeLayerCPUTestParamSet> obj) {
+        Precision netPrecision;
+        InputShape inputShapes;
+        std::vector<size_t> inputOrder;
+        std::string targetDevice;
+        CPUSpecificParams cpuParams;
+        std::map<std::string, std::string> additionalConfig;
+        std::tie(inputShapes, inputOrder, netPrecision, targetDevice, additionalConfig, cpuParams) = obj.param;
+
+        std::ostringstream result;
+        result << "IS=" << CommonTestUtils::partialShape2str({inputShapes.first}) << "_";
+        result << "TS=" << CommonTestUtils::vec2str(inputShapes.second) << "_";
```

Comment:
The second part of the `inputShapes` pair is a vector of vectors, so we should print it using loop
```cpp
        result << "TS=(";
        for (const auto& shape : inputShapes.second) {
            result << CommonTestUtils::vec2str(shape) << "_";
        }
        result << ")_";
```

Reply:
OK, I will change.
FYI :
 there is a vec2str overload which takes vector of vectors as input: 
```cpp
template<typename vecElementType>
inline std::string vec2str(const std::vector<std::vector<vecElementType>> &vec) {
    std::ostringstream result;
    for (const auto &v : vec) {
        result << vec2str<vecElementType>(v);
    }
    return result.str();
}
```

The only difference is it didn't use "_" to separate shapes and lack of outermost braces `()`.